src/OpenApi/sample/Endpoints/MapCircularSchemaEndpoints.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Sample.Endpoints;
+
+public static class CircularEndpointsExtensions
+{
+    public static IEndpointRouteBuilder MapCircularEndpoints1(this IEndpointRouteBuilder endpointRouteBuilder)
```

Comment:
Can we move this scenario to the schemas-by-ref endpoints? Are these endpoints being separate a key part of the scenario?